packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -5,55 +5,89 @@ import { FontSizes } from './AzureType';
 import * as StyleConstants from './Constants';
 
 const darkExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
+  calText: '#ffffff',
   bodyBackground: DarkSemanticColors.background,
   bodyText: DarkSemanticColors.text.body,
   bodyDivider: CommonSemanticColors.dividers.lineSeparator,
   buttonBackground: DarkSemanticColors.secondaryButton.rest.background,
   buttonBackgroundChecked: DarkSemanticColors.secondaryButton.pressed.background,
   buttonBackgroundCheckedHovered: DarkSemanticColors.secondaryButton.hover.background,
-  buttonBackgroundDisabled: DarkSemanticColors.disabledButton.background,
+  buttonBackgroundDisabled: 'rgb(50, 49, 48)',
   buttonBackgroundHovered: DarkSemanticColors.secondaryButton.hover.background,
-  buttonBackgroundPressed: DarkSemanticColors.secondaryButton.pressed.background,
-  buttonText: DarkSemanticColors.secondaryButton.rest.border,
+  buttonBackgroundHoveredDefault: 'rgb(37, 36, 35)',
+  buttonBackgroundPressed: 'rgb(41, 40, 39)',
+  buttonBorderColor: 'rgb(151, 150, 147)',
+  buttonBorderHovered: 'rgb(151, 150, 147)',
+  buttonText: '#ffffff',
   buttonTextChecked: DarkSemanticColors.secondaryButton.pressed.border,
   buttonTextCheckedHovered: DarkSemanticColors.secondaryButton.hover.border,
-  buttonTextDisabled: DarkSemanticColors.disabledButton.text,
+  buttonTextDisabled: 'rgb(121, 119, 117)',
   buttonTextHovered: DarkSemanticColors.secondaryButton.hover.border,
   buttonTextPressed: DarkSemanticColors.secondaryButton.pressed.border,
+  checkboxBorder: 'rgb(243, 242, 241)',
+  checkboxBorderHover: '',
+  checkboxBorderChecked: '',
+  checkmarkColorHover: 'rgb(161, 159, 157)',
+  checkboxBorder2: 'rgb(40, 153, 245)',
+  checkboxBorderDisabled: 'rgb(72, 70, 68)',
+  checkboxCheckedDisabled: 'rgb(72, 70, 68)',
+  checkboxCheckedMarkDisabled: '#000000',
+  checkboxIndeterminateBackground: 'transparent',
+  checkMarkFill: 'rgb(27, 26, 25)',
+  controlGroupBorder: 'rgb(243, 242, 241)',
+  checkBoxselectedBorderHover: 'rgb(108, 184, 246)',
+  groupChoiceFill: '#0078D4',
+  contextMenuBackground: '#000000',
   disabledBackground: CommonSemanticColors.backgrounds.disabled,
-  disabledBodyText: DarkSemanticColors.text.disabled,
+  disabledToggleBackground: 'transpanret',
```

Comment:
```suggestion
  disabledToggleBackground: 'transparent',
```

Reply:
Utilizing dark and light theme references instead of transparent.